scripts/import_standard_ebooks.py

```diff
@@ -0,0 +1,166 @@
+#! /usr/bin/env python
+import json
+import requests
+import sys
+import time
+from typing import Any, Dict, List, Optional
+
+import os.path as path
+
+import feedparser
+
+from openlibrary.core.imports import Batch
+
+sys.path.insert(0, path.abspath(path.join(path.sep, 'openlibrary')))
+from openlibrary.config import load_config  # noqa: E402
+load_config(
+    path.abspath(path.join(
+        path.sep, 'olsystem', 'etc', 'openlibrary.yml')))
+from infogami import config  # noqa: E402,F401
+
+FEED_URL = 'https://standardebooks.org/opds/all'
+LAST_UPDATED_TIME = './standard_ebooks_last_updated.txt'
+
+
+def get_feed():
+    """Fetches and returns Standard Ebook's feed."""
+    r = requests.get(FEED_URL)
+    return feedparser.parse(r.text)
+
+
+def map_data(entry) -> Dict[str, Any]:
```

Comment:
I don't believe you need to switch to `Dict` and `List` ; `list` and `dict` should work. You just can't use `Foo | None` ; the pipe operator.

Reply:
Without the `annotations` import, we can't define the types for a dict's keys and values, nor can we define the types of a list's items.  This signature would change to this:
```python
def map_data(entry) -> dict:
```
Does this match your expectations?

Reply:
:+1: Lgtm!